### DISCUSSION TITLE
Add image uploader to new post editor

### DISCUSSION THREAD

**Discussion by FinestAF** [NamelessMC/Nameless#2170] [Completed Ideas]
Please could you add a button that says 'upload image' when making a post so you can easily select and attach an image you want.

**Comment by samerton** [comment]
The old post editor had image uploading, I just need to investigate adding it to the new one.

**Comment by compi2k** [comment]
any updates on this?

thanks

**Reply by tadhgboyle** [reply]
If there we updates they'd be posted here.
This is a complicated thing to add to a 3rd party editor. 
PRs are open if you want to add it yourself until we get to it. Thanks!

**Comment by PadowYT2** [comment]
It is telling "You've reached the limit", it is actually uploading the image to `data://chAraCTers`, it must upload in `/uploads/` for example. How does CkEdtior handle this, maybe a function exists?

**Comment by Derkades** [comment]
Will be closed by #2653
